### PR TITLE
optimize: fix failed to get server recovery properties

### PR DESCRIPTION
**Pull request by tuwenlin**
fix:#4900

### CODE REVIEW

File: seata-spring-autoconfigure/seata-spring-autoconfigure-server/src/main/java/io/seata/spring/boot/autoconfigure/properties/server/ServerRecoveryProperties.java [hunk 1]
```diff
@@ -27,15 +27,44 @@
 @ConfigurationProperties(prefix = SERVER_RECOVERY_PREFIX)
 public class ServerRecoveryProperties {
 
-    private Integer handleAllSessionPeriod = 1000;
+    private Integer committingRetryPeriod = 1000;
```
Comment (slievrly) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-server/src/main/java/io/seata/spring/boot/autoconfigure/properties/server/ServerRecoveryProperties.java:30]: why Integer not int ?

Comment (tuwenlin) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-server/src/main/java/io/seata/spring/boot/autoconfigure/properties/server/ServerRecoveryProperties.java:30]: I just copy old properties,I will change it to int

**Review by a364176773 — APPROVED** [review]
LGTM

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**